lib/resources/r4/related_person_patch.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: Related Person
+field_name_base_url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson
+operations:
+  - name: add-relationship
+    path: /relationship/-
+    operation: add
+    type: CodeableConcept
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.relationship
+    description: The nature of the relationship between a patient and the related person.
+    example: |
+      [
+        {
+          "op": "add",
+          "path": "/relationship/-",
+          "value":{
+            "coding": [
+              {
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/351",
+                "code": "1152"
+              }
+            ]
+          }
+        }
+      ]
+
+    note: >
+      <ul>
+        <li>The <code>value.coding</code> must contain exactly one proprietary system.</li>
+        <li>The <code>value.coding</code> cannot contain a standard system.</li>
+        <li>Duplicate relationships will not be returned on a read.</li>
```

Comment:
how bout we say something like...

'Duplicate relationships may be added, but will only be represented once on the Resource.'

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/527/commits/7d6aadabc274f0e4e4a9ea1f037e6532c0c99abd#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR31